meta-lmp-base/recipes-core/initrdscripts/initramfs-module-install-efi/init-install-efi.sh

```diff
@@ -214,7 +214,19 @@ while true; do
         done
         break
     elif [ "$answer" = "n" ]; then
-        echo "Not erasing current partition table for device ${device}, assuming ${bootfs} as boot/ESP and ${rootfs} as rootfs."
+        echo "Not erasing current partition table for device ${device}. Use ${bootfs} as boot/ESP and ${rootfs} as rootfs? (y - default / n)"
+        echo
+        read answer
+        if [ "$answer" = "n" ]; then
+            echo "Please define the partition to be used as boot/ESP (e.g. ${bootfs}): "
+            echo
+            read answer
+            bootfs=${answer}
```

Comment:
given that the user can change the default value maybe is better to validate answer for `bootfs` and `rootfs`
```
while true; do
  echo "Please define the partition to be used as boot/ESP (e.g. ${bootfs}): "
  echo
  read answer
    if [ -b ${answer} ]; then
      break
    else
      echo "Invalid block device !"
    fi
done
```

Reply:
Had that initially but it kind of looks ugly to have a set of nested loops just to confirm when this option is already for advanced users.

But I agree that it is better to check, I'm sure people will make mistakes here.